doc/source/whatsnew/v1.3.0.rst

```diff
@@ -242,6 +242,7 @@ I/O
 - Bug in :func:`read_csv` not accepting ``usecols`` with different length than ``names`` for ``engine="python"`` (:issue:`16469`)
 - Bug in :func:`read_csv` raising ``TypeError`` when ``names`` and ``parse_dates`` is specified for ``engine="c"`` (:issue:`33699`)
 - Allow custom error values for parse_dates argument of :func:`read_sql`, :func:`read_sql_query` and :func:`read_sql_table` (:issue:`35185`)
+- Bug in :class:`Datetime64Formatter` that caused error on string representation with extension types of datetime64 values and ndim > 1 (:issue:`38390`)
```

Comment:
this is not user facing (as its internal), is there anything that is?

Reply:
The bug was found when calling `repr()` on an `ExtensionType`, I will put this in terms of that instead.